docarray/typing/tensor/abstract_tensor.py

```diff
@@ -116,3 +117,8 @@ def __docarray_from_native__(cls: Type[T], value: Any) -> T:
     def get_comp_backend() -> Type[AbstractComputationalBackend]:
         """The computational backend compatible with this tensor type."""
         ...
+
+    @abc.abstractmethod
+    def n_dim(self) -> int:
+        """Get number of tensor dimensions."""
+        ...
```

Comment:
not sure this is necessary, since (for now) tensors implement ndim in the same way. My concern is that we're adding an unnecessary method to the public interface. @JoanFM what do you think ?

Reply:
I agree, let's not add it here. If anything this should go into the computational backend. The tensor interface itself should stay as lean as possible, we don't want to run the risk of interfering with the "actual" tensors that we inherit from (torch, numpy, etc).